### PR TITLE
50 patch frontend layout

### LINKED ISSUE #50: Patch frontend layout
Current layout of our components on the frontend display a lot of unimplemented features which is fine.

However, our two main core features, **file upload + quiz**, are nested under sections that do not make much sense. 

Move them out to be under the dashboard component. 

NOTE: These changes will conflict with our "test procedures" section of the report, which is what Ouldooz will look at during our PI-2 meetings. Perhaps, we should demo two different versions of our frontend?

### PR DESCRIPTION
**Pull request by yenuGH**

### CODE REVIEW

**Review by jerrydngzh — APPROVED** [review]
lgtm

File: frontend/src/pages/home_page.tsx [hunk 1]
```diff
@@ -46,6 +48,53 @@ export function HomePage() {
         navigate(path)
     }
 
+    /* ------------------------------- components ------------------------------- */
+
+    function createNavLink(
```
Comment (toastjpg) [inline frontend/src/pages/home_page.tsx:53]: could extract to external components but we dont have the time lol

**Review by toastjpg — APPROVED** [review]
lgtm. Make sure to deploy + test all features

**Comment by yenuGH** [comment]
> lgtm. Make sure to deploy + test all features

no changes were made to the api gateway or the backend at all, just some frontend visual updates

**Comment by yenuGH** [comment]
not merging into main, will be merged into a presentation branch

**Pull request closed without merging**